tests/system-tests/internal/supporttools/tcpdump.go

```diff
@@ -0,0 +1,543 @@
+package supporttools
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/apiobjectshelper"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	scc "github.com/openshift-kni/eco-gotests/tests/system-tests/internal/scc"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	httpProtocol     = "http"
+	udpProtocol      = "udp"
+	tcpCaptureScript = `#!/bin/bash
+tcpdump -vvv -e -nn -i %s | egrep -i 'Host:'
+`
+
+	udpCaptureScript = `#!/bin/bash
+tcpdump -n -s0  -i %s port %s and udp
+`
+	logScanMaxTries = 3
+)
+
+// CreateTCPDumpDeployment creates support-tools tcpdump deployment on the nodes specified in scheduleOnNodes.
+func CreateTCPDumpDeployment(
+	apiClient *clients.Settings,
+	tdNamespace,
+	tdDeploymentLabel,
+	tdImage,
+	packetCaptureInterface,
+	packetCaptureProtocol string,
+	scheduleOnNodes []string) (*deployment.Builder, error) {
+	if packetCaptureProtocol != httpProtocol && packetCaptureProtocol != udpProtocol {
+		return nil, fmt.Errorf("CreateTraceRouteDeployment supports only %s and "+
```

Comment:
nit: method is _CreateTCPDumpDeployment_ but error mentions `CreateTraceRouteDeployment`:
```suggestion
		return nil, fmt.Errorf("CreateTCPDumpDeployment supports only %s and "+
```

Reply:
yep, thanks, I missed it